src/main/java/seedu/command/interpreter/PerformanceCommandInterpreter.java

```diff
@@ -1,9 +1,7 @@
 package seedu.command.interpreter;
 
 import seedu.command.Command;
-import seedu.command.performance.AddPerformance;
-import seedu.command.performance.DeletePerformance;
-import seedu.command.performance.ListStudentPerformance;
+import seedu.command.performance.*;
```

Comment:
Would this fail doing the checkstylemain?

Reply:
this is an outdated version